awx/main/models/credential/__init__.py

```diff
@@ -577,6 +577,16 @@ def askable_fields(self):
             if field.get('ask_at_runtime', False) is True
         ]
 
+    @property
+    def plugin(self):
+        if self.kind != 'external':
+            raise AttributeError(_('plugin'))
```

Comment:
Maybe don't wrap this in `i18n`, it's weird that you'd call `foo.plugin` and raise an `AttributeError` with plugin in french.

Reply:
😆 je suis d'accord avec toi